static/js/components/WebsiteCollectionItemsEditor.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback } from "react"
+import { useSelector } from "react-redux"
+import {
+  DndContext,
+  closestCenter,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors
+} from "@dnd-kit/core"
+import {
+  SortableContext,
+  sortableKeyboardCoordinates,
+  verticalListSortingStrategy
+} from "@dnd-kit/sortable"
+
+import { useMutation, useRequest } from "redux-query-react"
+import { WebsiteCollection } from "../types/website_collections"
+import {
+  editWebsiteCollectionItemMutation,
+  websiteCollectionItemsRequest,
+  createWebsiteCollectionItemMutation
+} from "../query-configs/website_collections"
+import { getWebsiteCollectionItemsCursor } from "../selectors/website_collections"
+import useAllWebsites from "../hooks/useAllWebsites"
+import WebsiteCollectionItemForm from "./forms/WebsiteCollectionItemForm"
+import { WCItemCreateFormFields } from "../types/forms"
+import SortableWebsiteCollectionItem from "./SortableWebsiteCollectionItem"
+
+interface Props {
+  websiteCollection: WebsiteCollection
+}
+
+export default function WebsiteCollectionItemsEditor(
+  props: Props
+): JSX.Element {
+  const { websiteCollection } = props
+
+  const sensors = useSensors(
+    useSensor(PointerSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates
+    })
+  )
+
+  useRequest(websiteCollectionItemsRequest(websiteCollection.id))
+
+  const websiteCollectionItemsCursor = useSelector(
+    getWebsiteCollectionItemsCursor
+  )
+
+  const items = websiteCollectionItemsCursor(websiteCollection.id)
+
+  const [, updateWCItemPosition] = useMutation(
+    editWebsiteCollectionItemMutation
+  )
+
+  function handleDragEnd(event: any) {
+    const { active, over } = event
+
+    if (active.id !== over.id) {
+      updateWCItemPosition(
+        {
+          position: items.map(item => String(item.id)).indexOf(over.id)
+        },
+        websiteCollection.id,
+        active.id
+      )
+    }
+  }
+
+  const [, createWCItem] = useMutation((values: WCItemCreateFormFields) =>
+    createWebsiteCollectionItemMutation(values, websiteCollection.id)
+  )
+
+  const onWCItemSubmit = useCallback(
+    (values: WCItemCreateFormFields) => {
+      createWCItem(values)
+    },
+    [createWCItem]
+  )
+
+  const websites = useAllWebsites()
+
+  return (
+    <div className="collection-item-editor pt-5 pb-3">
+      <h4>Courses in this collection</h4>
```

Comment:
One quick thing I noticed

```suggestion
      <h4>Sites in this collection</h4>
```

Reply:
@Ferdi wondering your thoughts on this, I made this header 'Courses in this collection' following what was in the mockups, should we stick with that or go with the more general 'Sites in this collection' ?